python/pyspark/sql/connect/session.py

```diff
@@ -694,14 +694,10 @@ def streams(self) -> "StreamingQueryManager":
     streams.__doc__ = PySparkSession.streams.__doc__
 
     def __getattr__(self, name: str) -> Any:
-        if name in ["_jsc", "_jconf", "_jvm", "_jsparkSession"]:
+        if name in ["_jsc", "_jconf", "_jvm", "_jsparkSession", "sparkContext", "newSession"]:
             raise PySparkAttributeError(
                 error_class="JVM_ATTRIBUTE_NOT_SUPPORTED", message_parameters={"attr_name": name}
```

Comment:
I see this in the error message:`If you need to use this attribute, do not use Spark Connect when creating your session.`
Could we provide more detailed instructions on how users can "not use" Spark Connect?

Reply:
Sounds good. Added link for creating regular Spark Session docs in the error message:

```
Visit https://spark.apache.org/docs/latest/sql-getting-started.html#starting-point-sparksession for creating regular Spark Session in detail.
```